site/_docs/continuous-integration.md

```diff
@@ -118,6 +118,7 @@ customizable. If your script won't change much, you can write your test
 incantation here directly:
 
 {% highlight yaml %}
+before_install: gem install jekyll html-proofer
```

Comment:
I personally think that the line should mention something to the effect of what you just implied to me (if I am interpreting right) because a Gemfile is definitely the Ruby convention now days.